src/main/java/subway/station/StationReader.java

```diff
@@ -8,9 +8,9 @@
 public class StationReader {
   private final StationRepository stationRepository;
 
-  public Station read(Long id) {
+  public Station readById(Long id) {
     return stationRepository
         .findById(id)
-        .orElseThrow(() -> new IllegalArgumentException("해당 역이 존재하지 않습니다."));
+        .orElseThrow(() -> new StationNotFoundException("역 #" + id + "이 존재하지 않습니다."));
```

Comment:
예외 메시지 내용은 예외 클래스 내부로 옮겨보면 어떨까요?
```suggestion
        .orElseThrow(() -> new StationNotFoundException(id));
```